docs/user-guide/_install-cli-via-proxy.md

```diff
@@ -2,6 +2,9 @@
 ​
```

Comment:
Note: No need to review this file. I'm removing the file and the content has been relocated.